testprojects/src/python/mypy/BUILD

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Instead of putting it in testprojects, create a new folder in `contrib/mypy/examples/src/python`.